app/Server.ts

```diff
@@ -18,13 +18,13 @@ export class Server {
 
   constructor(app: Express) {
     this.app = app
-    this.app.use(limiter)
     // Enable once we're behind a reverse proxy (Heroku, Bluemix, AWS ELB, Nginx, etc)
     //  see https://expressjs.com/en/guide/behind-proxies.html
     //  this.app.set('trust proxy', 1);
 
     this.app.use(express.static(path.resolve('./') + '/build/frontend'))
 
+    this.app.use(limiter)
```

Comment:
Need to use limiter after any static data is rendered, but before any other endpoints are defined that need it.